tests/routes/test_geo_stats.py

```diff
@@ -1,6 +1,7 @@
 from http.client import NOT_FOUND, OK
 from unittest.mock import Mock
-from app.initial_data import populate_initial_data
+from db_client.initial_data import populate_initial_data
+from app.core.security import get_password_hash
```

Comment:
Sidenote that it doesn't look like Ruff/Black has been configured to run the way we expect in your IDE?

`If you're using the [VS Code Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python), you can configure VS Code to fix violations on-save using Ruff, then re-format with [the Black extension](https://marketplace.visualstudio.com/items?itemName=ms-python.black-formatter), via the following settings.json:`

```json

    "[python]": {
       "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.fixAll": "true|explicit",
            "source.organizeImports": "true|explicit"
        },
        "editor.defaultFormatter": "ms-python.black-formatter",
    },
```